starknet_py/net/account/account_proxy.py

```diff
@@ -0,0 +1,125 @@
+from typing import List, Optional, Union
+
+from starknet_py.net import AccountClient
+from starknet_py.net.account.base_account import BaseAccount
+from starknet_py.net.client import Client
+from starknet_py.net.client_models import (
+    Calls,
+    SentTransactionResponse,
+    Declare,
+    InvokeFunction,
+    EstimatedFee,
+    Hash,
+    Tag,
+)
+from starknet_py.net.models import AddressRepresentation
+from starknet_py.net.models.transaction import DeployAccount
+from starknet_py.net.models.typed_data import TypedData
+
+
+class _AccountProxy(BaseAccount):
```

Comment:
This is a temporary solution for using old `AccountClient` in code that expects new Account (to avoid multiple implementation in the transition period).